spec/operators/audit-spec.ts

```diff
@@ -1,315 +1,388 @@
 import { expect } from 'chai';
+import { TestScheduler } from 'rxjs/testing';
 import { of, interval, EMPTY } from 'rxjs';
 import { audit, take, mergeMap } from 'rxjs/operators';
-import { hot, cold, expectObservable, expectSubscriptions } from '../helpers/marble-testing';
+import { observableMatcher } from '../helpers/observableMatcher';
 
 declare function asDiagram(arg: string): Function;
 
 /** @test {audit} */
 describe('audit operator', () => {
+  let testScheduler: TestScheduler;
+
+  beforeEach(() => {
+    testScheduler = new TestScheduler(observableMatcher);
+  });
+
   asDiagram('audit')('should emit the last value in each time window', () => {
-    const e1 =   hot('-a-xy-----b--x--cxxx-|');
-    const e1subs =   '^                    !';
-    const e2 =  cold( '----|                ');
-    const e2subs =  [' ^   !                ',
-                   '          ^   !       ',
-                   '                ^   ! '];
-    const expected = '-----y--------x-----x|';
-
-    const result = e1.pipe(audit(() => e2));
-
-    expectObservable(result).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
-    expectSubscriptions(e2.subscriptions).toBe(e2subs);
+    testScheduler.run(({ hot, cold, expectObservable, expectSubscriptions }) => {
+      const e1 = hot('    -a-xy-----b--x--cxxx-|');
+      const e1subs = '    ^--------------------!';
+      const e2 = cold('    ----|                ');
+      const e2subs = [
+        '                 -^---!                ',
+        '                ----------^---!        ',
+        '                ----------------^---!  '
+      ];
```

Comment:
Should the `e2subs[1]` and `e2subs[2]` start a tick before all the other marbles like this? (actually i think previously it was 2 ticks - so I'll fix one way or the other)